content/en/docs/refguide/modeling/application-logic/workflows/boundary-events.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Boundary Events"
+url: /refguide/boundary-events/
+weight: 10
+---
+
+{{% alert color="info" %}}
+This feature is currently in public beta as of version 10.15. Please note that certain features or attributes may be subject to change and may contain bugs.{{% /alert %}}
+
+## Introduction
+
+* [BPMN 2.0](#BPMN-2.0)
+* [Types](#types)
+* [Supporting Activities](#supporting-activities)
+* [Getting started](#getting-started)
+* [Adding Timer Boundary Events](#adding-timer-boundary-events)
+* [Rearranging](#rearranging)
+* [Execution](#execution)
+* [Current limitations](#current-limitations)
+* [Read More](#read-more)
+
+## BMPN 2.0
+
+In BPMN 2.0, boundary events are a type of event that is attached to the boundary of an activity (such as a task or subprocess) to handle exceptional situations or are triggered by certain behaviours. The icon inside the event indicates the type of event (e.g. clock indicates it’s a timer event). Boundary Events are always displayed by 2 circles (either solid or dashed) and are linked by a dotted line to the parent activity. 
+
+There are two main types of boundary events:
+
+- Non-Interrupting Boundary Events: These events do not interrupt the ongoing activity. When triggered, they allow the activity to continue while simultaneously starting a new path from the boundary event. As per BPMN2.0 specification, non-interrupting boundary events are visualised as two dashed circles with an icon in the center.
+
+- Interrupting Boundary Events: When these events are triggered, they interrupt the normal path of the activity they are attached to. The activity stops, and the process flow is redirected to the boundary event's outgoing sequence path. As per BPMN2.0 specification, interrupting boundary events are visualised as two solid circles.
```

Comment:
pls add that interrupting not part of beta.

Reply:
Added extra line which explains this.